examples/e05_command_framework/src/main.rs

```diff
@@ -307,6 +307,13 @@ async fn main() {
     let mut client = Client::builder(&token)
         .event_handler(Handler)
         .framework(framework)
+        // For this example to run properly, the "Presence Intent" and "Server Members Intent" 
+        // options need to be enabled.
+        // These are needed so the `required_permissions` macro works on the commands that need to
+        // use it.
+        // You will need to enable this 2 options on the bot application, and possibly wait up to 5
```

Comment:
The [determiner](https://en.wikipedia.org/wiki/Determiner) is applied on a noun in plural form, thus it also has to be plural.
```suggestion
        // You will need to enable these 2 options on the bot application, and possibly wait up to 5
```